index.html

```diff
@@ -0,0 +1,781 @@
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail3.webp" alt="Dragon Capsule on Parachute" width="250"
+                                height="180">
+                            <figcaption class="gray very-small-text "> <a href="#">Bill Ingalls/NASA, via Agence
+                                    France-Press - Getty Images</a></figcaption>
+                        </figure>
+                        <p><a href="#">'Thanks for Flying SpaceX': NASA Astronauts Safely Splash Down after
+                                Journey From Orbit
+                            </a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 2</span>
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail4.webp" alt="Star going Supernova" width="250" height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>L. Calcada/ESO</figcaption>
+                        </figure>
+                        <p><a href="#">A Star Went Supernova in 1987. Where Is It Now?</a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 8</span>
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail5.webp" alt="Rocket lifts off from the Launch pad" width="250"
+                                height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>Image By Joe Skiper/Reuters</figcaption>
+                        </figure>
+                        <p><a href="#">NASA Launches Perseverance Rover, Capping Summer of Missions to Mars</a>
+                            <br>
+                            <span class="gray very-small-text">July. 30</span>
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail6.webp" alt="Hole on Mars" width="250" height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>NASA/JPL-Caltech/University of Arizona</figcaption>
+                        </figure>
+                        <p><a href="#">How NASA Found the Ideal Hole on Mars to Land In</a>
+                            <br>
+                            <span class="gray very-small-text">July. 30</span>
+                        </p>
+                    </div>
+                </div>
+            </section>
```

Comment:
- [ ] Also the bottom section is not centered as it is on the original page. Please fix it.

![image](https://user-images.githubusercontent.com/60184961/91614969-3ceb4900-e98b-11ea-875d-5c7440436e05.png)

Reply:
This has been resolved